src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/treewalker/InputTreeWalkerSkipParsingExceptionConfig.java

```diff
@@ -0,0 +1,12 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <property name="skipFileOnJavaParseException" value="true"/>
+    <property name="javaParseExceptionSeverity" value="ignore"/>
+    <module name="ConstantName"/>
```

Comment:
Add here check that participate in diff report.

Reply:
NoCodeInFile

Reply:
I have finished these updates. Waiting for #14852 to be solved, so we can test that with this PR.

Reply:
Performance test  PR is not related.
Ok Let keep, `name="ConstantName"/>`  , but please create new test for NoCodeInFile, to show how it works.

Reply:
There are already some tests for NoCodeInFile.
For example, in the following test we do not expect any violations from NoCodeInFile if there is a java parsing exception.
```java
public void testSkipFileOnJavaParseExceptionSkipChecks() throws Exception {
        final DefaultConfiguration config = createModuleConfig(TreeWalker.class);
        config.addProperty("skipFileOnJavaParseException", "true");
        config.addProperty("javaParseExceptionSeverity", "ignore");
        config.addChild(createModuleConfig(NoCodeInFileCheck.class));

        final Checker checker = createChecker(config);

        final File[] files = {
            new File(getNonCompilablePath("InputTreeWalkerSkipParsingException.java")),
            new File(getPath("InputTreeWalkerProperFileExtension.java")),
        };
        final Map<String, List<String>> expectedViolation = new HashMap<>();
        expectedViolation.put(getPath("InputTreeWalkerProperFileExtension.java"),
                new ArrayList<>());

        verify(checker, files, expectedViolation);
    }
```